src/components/NegotiationList.vue

```diff
@@ -0,0 +1,217 @@
+<template>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+  <header>
+    <h3>
+      List of your Negotiations:
+    </h3>
+  </header>
+    <div class="float-child">
+    <table id="tableComponent" class="table table-hover styled-table">
+        <thead>
+        <tr>
+            <th  v-for="field in headers" :key='field' >
+                {{field}}
+            </th>
+            <th/>
+        </tr>
+        </thead>
+        <tbody v-for="item in negotiations" :key='item'>
+        <tr class="active-row">
+            <td>{{ item.id }}</td>
+            <td>{{ item.payload.project.title }}</td>
+            <td>{{ item.status }}</td>
+            <td>
+                <span style="white-space: pre">
+                    <button type="button" class="editButton" @click="interactModal(item)">
+                        <i class="fa fa-pencil"></i>
+                    Interact
+                </button>
+                <button class="abandonButton" @click="abandonRequest">
+                    <i class="fa fa-trash"></i>
+                    Abandon
+                </button>
+                </span>
+            </td>
+        </tr>
+        </tbody>
+    </table>
+    </div>
+    <div class="float-child">
+        <h6 class="bg-light p-2 border-top border-bottom filter-header">Filter</h6>
+        <ul class="list-group border-2">
+            <li class="list-group-item list-group-item-action">All <span class="badge badge-primary rounded-pill">2</span></li>
+            <li class="list-group-item list-group-item-action">Year 2022 <span class="badge badge-primary rounded-pill">0</span></li>
+            <li class="list-group-item list-group-item-action">Accepted <span class="badge badge-primary rounded-pill">0</span></li>
+        </ul>
+    </div>
+    <div v-if="showModal" class="modal">
```

Comment:
What if we create Negotiation's page instead of Modal?

Reply:
UX team like the idea of  modal, this is a first implementation. Their feedback should respond to most of your comments